src/studio/nuxeo-studio/tutorials/how-to-work-with-git-and-studio-project.md

```diff
@@ -0,0 +1,103 @@
+{{/callout}}
+{{#> callout type='warning' heading='Hot reload'}}
+Your studio project should be hot-reloaded in your server once at least to load the Nuxeo Studio project web resources
+{{/callout}}
+
+# Setup your dev environment
+
+- Open https://connect.nuxeo.com/ and navigate to **My Applications** tab
+- Find your project and copy the git URL
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/Studio/How to Work with Git and Your Nuxeo Studio Project/Studio_Git_URL.png
+    name: Studio_Git_URL.png
+    studio_designer#screenshot#up_to_date
+--}}
+![Studio_Git_URL.png](nx_asset://967edd4f-47c6-4635-a206-f0e8b811f34f ?w=650,border=true)
+
+- Create a new directory in your local workspace
+- Clone the Studio project
+  - `git clone https://connect.nuxeo.com/nuxeo/git/my-studio-project.git` for example
+- Synchronize your git project with your Nuxeo server
+  - `nuxeo sync --src <path of your project> --dest <path to the UI folder of your nuxeo server instance>`
+  - Example: `nuxeo sync --src university-SANDBOX/studio/resources/nuxeo.war/ui/ --dest nuxeo-server-10.2-tomcat/nxserver/nuxeo.war/ui/`
+
+You should read the following lines in your command line editor:
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/Studio/How to Work with Git and Your Nuxeo Studio Project/Nuxeo_sync_result.png
+    name: Nuxeo_sync_result.png
+    studio_modeler#screenshot#up_to_date
+--}}
+![Nuxeo_sync_result.png](nx_asset://078d268f-8f85-42b5-86ca-f2f67cb03893 ?w=650,border=true)
+
+All the actions executed on your local project (creation, edition, deletion) are instantly replicated to your local Nuxeo server. To load your modification, all you need to do is to refresh the Web UI webpage to visualize your changes.
+
+# Push your contribution into your Studio project
+
+When you've finished your project, you can commit and push your changes back to your Nuxeo Studio Project.
+
+A standard development flow with Studio Designer can be:
+- Create a development branch from Nuxeo Studio
+  - `feature/new-admin-interface` for example
+- Build some Modeler contribution which are needed for your Designer project
+  - Typically to create a page provider
+- Get your Studio project locally so that you can work on the Designer contribution
+
+```
+git clone https://connect.nuxeo.com/nuxeo/git/my-studio-project.git
+git pull feature/new-admin-interface
+```
+
+- Create your custom code. Once finished, push your contribution
+
+```
+git add -A
+git commit -m 'New Admin Interface'
+git push
+```
+
+- Go back to your Studio Project on https://connect.nuxeo.com and integrate your changes in your Studio Project by clicking the icon ![]({{file name='pull.png' page='studio-icons-index'}} ?w=32) so that it can be available for all project contributors, and create the Studio marketplace package
```

Comment:
Use bullets here:
* Go back to Nuxeo Studio
* Click the icon xxx to integrate your changes (the contribution will be shared between all the contributors)
Not sure the rest is necessary, we know how create a marketplace contribution (a link to another tutorial)?